recipes/ace/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export ACE_ROOT=$SRC_DIR
+
+if [ `uname` == Darwin ]; then
+    # How do we know what system we are on?
+    if [ ${MACOSX_DEPLOYMENT_TARGET} = '10.9' ]; then
+        echo "#include \"ace/config-macosx-mavericks.h\"" > ace/config.h
+        echo "include \$(ACE_ROOT)/include/makeinclude/platform_macosx_mavericks.GNU" > include/makeinclude/platform_macros.GNU
+    elif [ ${MACOSX_DEPLOYMENT_TARGET} = '10.10' ]; then
+        echo "#include \"ace/config-macosx-yosemite.h\"" > ace/config.h
+        echo "include \$(ACE_ROOT)/include/makeinclude/platform_macosx_yosemite.GNU" > include/makeinclude/platform_macros.GNU
+    elif [ ${MACOSX_DEPLOYMENT_TARGET} = '10.11' ]; then
+        echo "#include \"ace/config-macosx-elcapitan.h\"" > ace/config.h
+        echo "include \$(ACE_ROOT)/include/makeinclude/platform_macosx_elcapitan.GNU" > include/makeinclude/platform_macros.GNU
+    fi
+fi
+
+if [ `uname` == Linux ]; then
+    echo "#include \"ace/config-linux.h\"" > ace/config.h
+    echo "include \$(ACE_ROOT)/include/makeinclude/platform_linux.GNU" > include/makeinclude/platform_macros.GNU
```

Comment:
Same story.

Reply:
I am not very familiar with this package, so maybe there is a better way.  What I did here was attempt to perform steps 3 and 4 of the Unix install instructions:
http://www.dre.vanderbilt.edu/~schmidt/DOC_ROOT/ACE/ACE-INSTALL.html#unix